README.md

```diff
@@ -1,6 +1,58 @@
 # byconity.github.io
 
-## Content Folders
+## Developing
+
+This website is powered by [Docusarus](https://docusaurus.io/).
+You are encouraged to understand the basics of Docusaurus first.
+
+```bash
+pnpm install # Install NPM dependencies
```

Comment:
Normally we are using npm by default, please add some introduction of pnpm before introducing it.